app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.